build.gradle

```diff
@@ -2862,26 +2771,6 @@ project(':streams:integration-tests') {
 
     testRuntimeOnly runtimeTestLibs
   }
-
-  sourceSets {
```

Comment:
please don't remove this. we should remove the sourceSets which is same to default value.